scripts/breed_bird_survey_50stop.py

```diff
@@ -14,9 +14,13 @@
 import zipfile
 from decimal import Decimal
 from retriever.lib.templates import Script
-from retriever.lib.models import Table, Cleanup, no_cleanup, correct_invalid_value
-from retriever import HOME_DIR, open_fr, open_fw, VERSION
+from retriever.lib.models import Table, Cleanup, correct_invalid_value
 from pkg_resources import parse_version
+try:
+    from retriever import HOME_DIR, open_fr, open_fw, VERSION
+except ImportError:
+    from retriever.lib.scripts import open_fr, open_fw
+    from retriever.lib.defaults import VERSION
```

Comment:
The try/except contents are backwards compared to other datasets.

Reply:
done